source/installation/install-on-a-local-machine/install-on-a-local-machine.txt

```diff
@@ -71,7 +71,7 @@ Learn More
    :hidden:
    :titlesonly:
 
-   /installation/install-on-a-local-machine/install-windows
-   /installation/install-on-a-local-machine/install-mac
-   /installation/install-on-a-local-machine/install-ubuntu
-   /installation/install-on-a-local-machine/install-rhel
+   Windows </installation/install-on-a-local-machine/install-windows>
+   Mac </installation/install-on-a-local-machine/install-mac>
```

Comment:
[Suggestion]: As of a few releases ago, Apple now styles their operating system as `macOS`, (q.e.d., [Wiki](https://en.wikipedia.org/wiki/MacOS)).

Reply:
Done